bin/compile

```diff
@@ -384,6 +384,11 @@ case $TOOL in
     ;;
 esac
 
+if [ ! -z ${name+x} ];
+then
+  echo -e "---\ndefault_process_types:\n  web: $(basename $name)" >> /tmp/buildpack-release-step.yml
```

Comment:
This assumes $(basename $name) is the executable. This only works when $name is also the main package. I find this is rarely the case in anything except smaller apps.